src/OrchardCore.Modules/OrchardCore.Autoroute/Services/AutorouteEntries.cs

```diff
@@ -36,7 +36,7 @@ public void AddEntries(IEnumerable<AutorouteEntry> entries)
                         _contentItemIds.Remove(previousPath);
                     }
 
-                    var requestPath = "/" + entry.Path.TrimStart('/');
+                    var requestPath = "/" + entry.Path.Trim('/');
```

Comment:
Maybe better to use `.Trim(' ', '/')`, maybe not for perf and knowing that he admin UI disallows spaces in the path.